version/version.go

```diff
@@ -37,20 +47,44 @@ func Semver() string {
 	return version
 }
 
-// Println prints all available details about the current version in a
-// human-readable format, with an optional component name as the prefix
-func Println(component string) {
+func SemverWithEditionSuffix() string {
+	var editionSuffix string
+	switch Edition {
+	case "community":
+		editionSuffix = CommunityEditionSuffix
+	case "enterprise":
+		editionSuffix = EnterpriseEditionSuffix
+	default:
+		editionSuffix = InvalidEditionSuffix
+	}
+	return fmt.Sprintf("%s+%s", Semver(), editionSuffix)
+}
+
+func EditionOutput() string {
+	if Edition == "community" || Edition == "enterprise" {
+		return fmt.Sprintf("%s edition", Edition)
+	}
+	return fmt.Sprintf("built with an invalid \"edition\" ldflag")
```

Comment:
Why do we need this `invalid` edition and this logic?

Reply:
The value should only ever be set to community or enterprise; any other value will be a violation of the trademark guidelines. We will eventually be including the url to the trademark guidelines in the output if an invalid edition is specified.